backend/functions/consolidations/consolidations.function.test.ts

```diff
@@ -0,0 +1,95 @@
+import httpTrigger from './consolidations.function';
+import { MockData } from '../../../common/src/cams/test-utilities/mock-data';
+
+const context = require('azure-function-context-mock');
+
+const rejectConsolidation = jest.fn();
+const approveConsolidation = jest.fn();
+
+jest.mock('../lib/controllers/orders/orders.controller', () => {
+  return {
+    OrdersController: jest.fn().mockImplementation(() => {
+      return {
+        rejectConsolidation,
+        approveConsolidation,
+      };
+    }),
+  };
+});
+
+describe('Consolidations Function tests', () => {
+  test('should reject consolidation when procedure == "reject"', async () => {
+    const mockConsolidationOrder = MockData.getConsolidationOrder();
+    rejectConsolidation.mockImplementation(() => {
+      return Promise.resolve({ success: true, body: [mockConsolidationOrder] });
+    });
+    const request = {
+      params: {
+        procedure: 'reject',
+      },
+      body: {
+        ...mockConsolidationOrder,
+        rejectedCases: [mockConsolidationOrder.childCases[0]],
+      },
+      method: 'PATCH',
+    };
+    const expectedResponseBody = {
+      success: true,
+      body: [mockConsolidationOrder],
+    };
+    process.env = {
+      DATABASE_MOCK: 'true',
+    };
+    await httpTrigger(context, request);
+    expect(context.res.body).toEqual(expectedResponseBody);
+  });
+
+  test('should reject consolidation when procedure == "Approve"', async () => {
+    const mockConsolidationOrder = [MockData.getConsolidationOrder()];
+    approveConsolidation.mockImplementation(() => {
+      return Promise.resolve({ success: true, body: [mockConsolidationOrder] });
+    });
+    const request = {
+      params: {
+        procedure: 'approve',
+      },
+      method: 'PATCH',
+    };
+    const expectedResponseBody = {
+      success: true,
+      body: [mockConsolidationOrder],
+    };
+    process.env = {
+      DATABASE_MOCK: 'true',
+    };
+    await httpTrigger(context, request);
+    expect(context.res.body).toEqual(expectedResponseBody);
```

Comment:
Because of the way we are mocking the controller, I did not find a way to spy on the function, but I've modified the default behavior to throw an error so not catching an error ensures that the function we're not trying to test has not been called.

```
const rejectConsolidation = jest
  .fn()
  .mockRejectedValue('Set up the desired behavior for your test.');
const approveConsolidation = jest
  .fn()
  .mockRejectedValue('Set up the desired behavior for your test.');
```